app/views/user/_tracks_progress.erb

```diff
@@ -0,0 +1,22 @@
+<% if user_tracks_summary.any? %>
+  <h3> Progess </h3>
+  <% if params[:all] == 'true' %>
```

Comment:
What do we want to be the default in this case? Depending on the case we can change the way we are dealing with the params.